funnel/models/event.py

```diff
@@ -71,6 +72,7 @@ class Event(ScopedNameTitleMixin, db.Model):
     parent = db.synonym('proposal_space')
     ticket_types = db.relationship('TicketType', secondary=event_ticket_type)
     participants = db.relationship('Participant', secondary='attendee', backref='events', lazy='dynamic')
+    badge_template = db.Column(db.UnicodeText, nullable=True, default=u'')
```

Comment:
This is a URL, right? It should be `Unicode(250)` not `UnicodeText`. Also, don't mix nullable with empty string. Either nullable should be False, or no default should be provided.